### ISSUE TITLE
multiline titles and unicode for repo description

### ISSUE THREAD

**Issue by eshellman** [gutenbergbooks/git#3]
This may be only a cosmetic issue, but worth testing:

try initializing repos for #4576 "粉妝樓41-50回"

and #14280  "Holidays at Roselands&#13;
A Sequel to Elsie Dinsmore"

**Comment by acabal** [comment]
Yeah we need to improve how the script detects the title and author.  I'm sure there's a ton of cases where it will either fail or detect something wrong.  If you have a suggestion on how to improve title/author detection I'm all ears.

**Comment by acabal** [comment]
Unicode titles work fine, but multiline titles fail.  Instead of adding a bunch of edge cases to an already weak method of getting titles/authors, we should think of an alternative scheme.  (Or, we could just dispense with the title/author in the repo description, too.)

**Comment by eshellman** [comment]
maybe just don't fail the repo creation? you're only using these for the github description, which we could fix up without touching the pg servers.

**Comment by acabal** [comment]
Yeah, but out of 10,000+ repos even fixing 500 by hand would be a huge pain...

**Comment by eshellman** [comment]
I expect to touch 50,000 to add metadata files. wouldn't be much extra to reset the repo description.
